src/tests/utils/constants.cairo

```diff
@@ -0,0 +1,50 @@
+mod Constants {
```

Comment:
Could we name this to something more descriptive like `factory_constants`? We are going to have multiple external people watching our code and longer but more descriptive names are going to be useful.